plugins/flytekit-aws-sagemaker/flytekitplugins/awssagemaker_inference/workflow.py

```diff
@@ -105,7 +110,36 @@ def create_sagemaker_deployment(
                 if param not in wf.inputs.keys():
                     wf.add_workflow_input(param, t)
                 input_dict[param] = wf.inputs[param]
-        node = wf.add_entity(obj, **input_dict)
+        task_input_dict.append((obj, input_dict))
+
+    if override:
+        delete_sagemaker_deployment_wf = delete_sagemaker_deployment(
```

Comment:
will this error out if the model doesn't exist?

Reply:
yes, it'll error out.